src/hooks/useDebouncedFetchSections.ts

```diff
@@ -83,7 +83,7 @@ const useDebouncedFetchSection = (
         try {
           const response = await cioClient?.autocomplete.getAutocompleteResults(
             debouncedSearchTerm,
-            autocompleteParameters
+            { ...autocompleteParameters, ...advancedParameters.autocompleteParameters }
```

Comment:
as a maintainer of this lib three months from now, I'd find this to be very confusing

instead of this:
`{ ...autocompleteParameters, ...advancedParameters.autocompleteParameters }`

I'd prefer this: 
{ ...autocompleteParameters }

Reply:
👍 Had to move some code around but this sounds good to me.